test/acceptance/cli-test.js

```diff
@@ -614,9 +608,9 @@ describe('ember-template-lint executable', function () {
           'app/templates/application.hbs',
           '  1:4  error  Non-translated string used  no-bare-strings',
           '  1:24  error  Non-translated string used  no-bare-strings',
-          '  1:53  warning  HTML comment detected  no-html-comments',
+          '  1:53  error  HTML comment detected  no-html-comments',
```

Comment:
The title of this test `should print properly formatted error and warning messages` is outdated now?

Reply:
Ya I'll change the name.